File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        y=0
        k=x
        if x<0:
            return False
        while x>0:
            y=y*10+x%10
            x=x//10
        return k==y
        
        

            
             
        

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) 
    {
        char[] arr=String.valueOf(x).toCharArray(); 
       int l=0;
       int r=arr.length-1;
      
       while(l<r)
       {
          if(arr[l]==arr[r])
          {
            l++;
            r--;
          }
          else
          {
            return false;
          }
       }
       return true;
    }
}